manim/utils/hashing.py

```diff
@@ -73,9 +70,7 @@ def layer(func):
 
     @classmethod
     def check_already_processed(cls, obj: Any) -> Any:
-        """Checks if obj has been already processed. Returns itself if it has not been,
-        or the value of _ALREADY_PROCESSED_PLACEHOLDER if it has.
-        Marks the object as processed in the second case.
+        """Check if obj has been already processed. Returns itself if it has not been, or the value of _ALREADY_PROCESSED_PLACEHOLDER if it has. Marks the object as processed in the second case.
```

Comment:
```suggestion
        """If obj has not been processed, returns obj and marks it as processed. Otherwise returns _ALREADY_PROCESSED_PLACEHOLDER
```

The original doc refers to "second case," but it seems like it should be "first case."

Line still doesn't seem short enough